.travis.yml

```diff
@@ -94,6 +80,23 @@ matrix:
           - libavresample4
           - libswresample3
```

Comment:
Because Travis plays favorites, tending to kick off matrix jobs in the order they're listed in the Travis configs, having our _least_ critical (and quickest) build listed **first** was causing unnecessary delays even _starting_ the later, more-critical ones.

Now, the Coverage build is listed first (because it requires extra processing time to generate and upload the report data), then the FFmpeg 4 build, followed by FFmpeg3 (GCC, Clang), and finally FFmpeg 2. Because nobody really cares that much anymore, whether we can still build libopenshot with FFmpeg 2.